vdyp-fip/src/main/java/ca/bc/gov/nrs/vdyp/fip/FipStart.java

```diff
@@ -275,6 +275,22 @@ private void checkPolygon(FipPolygon polygon) throws ProcessingException {
 					polygon.getPolygonIdentifier(), Layer.PRIMARY, primaryLayer.getSiteIndex(), 0.5f
 			);
 		}
+
+		// TODO fill in "FR" should be a map of species to volume proportions for the
+		// primary layer, see fip_chk.for:237
+
+		for (var layer : polygon.getLayers().values()) {
+			var percentTotal = layer.getSpecies().values().stream()//
+					.map(FipSpecies::getPercentGenus)//
+					.reduce(0.0f, (x, y) -> x + y);
+			if (Math.abs(percentTotal - 100f) > 0.01f) {
+				throw validationError(
+						"Polygon %s has %s layer where species entries have a percentage total that does not sum to 100%%.",
```

Comment:
In theory, this test is actually testing that the percentage is greater than 100 and thus the Validation Error should state that the sum > 100%.